dev/run

```diff
@@ -504,6 +512,119 @@ def boot_nouveau(ctx):
     )
 
 
+CLOUSEAU_DIR = "clouseau"
+JAVA_VERSION_RE = re.compile('"(\d+\.\d+).*"')
+
+
+def get_java_version(java):
+    output = sp.check_output([java, "-version"], stderr=sp.STDOUT).decode("utf-8")
+    if output:
+        matches = JAVA_VERSION_RE.search(output)
+        if matches:
+            return float(matches.groups()[0])
+
+
+@log("Start Clouseau node clouseau{idx}")
+def boot_clouseau(ctx, idx):
+    def warn(reason):
+        print("Warning: " + reason)
+
+    if not os.path.isdir(CLOUSEAU_DIR):
+        warn(
+            "Clouseau deployment cannot be found, please run ./configure with --enable-clouseau"
```

Comment:
Shall we highlight the command?
"Clouseau deployment cannot be found, please run `./configure --enable-clouseau`"

Reply:
Sure.  Good idea.